DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html

```diff
@@ -1,12 +1,78 @@
 ﻿<!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Forget Password Token</title>
-</head>
-<body>
-    <h3>Use this token to reset your password.</h3>
-    <p><strong>Token :</strong> {body}</p>
-</body>
+    <title>Password Reset</title>
```

Comment:
Reset password.
The filename also should be reset password.

Reply:
update file name